### DISCUSSION TITLE
Opentrack not working on F1 24 and Flightgear on my PC

### DISCUSSION THREAD

**Discussion by Gelbbrustara** [opentrack/opentrack#1882] [Q&A]
Hello, I recently got F1 24 from Steam.

I really wanted to try out the eye tracking function, so I got "AI-Track" and "Opentrack". The tracking is also recognized really well via my webcam and I (think) that all the settings are set correctly. But when I open the game and go to the settings and select "Eye and Head Tracking" I can't activate the function for some reason?

It doesn't work properly with the flight simulator "Flightgear" either.

It would be great if you could help me :)

![Screenshot 2024-06-30 132926](https://github.com/opentrack/opentrack/assets/106803549/05176229-6a9d-4ac7-91e2-5d9b4a0cde31)

**Comment by velicast** [comment]
Same situation. I cannot enable the Extended View while OpenTrack+EyeBeam are already running.

**Comment by velicast** [comment]
I found the solution. You don't need AI Track. Just OpenTrack with neuralnet tracker. In the configuration of freetrack 2.0 Enhanced select Enable TrackIR and disable freetrack. Then in F1 23/24, start the race, select cockpit camera, then go to Eye and Head tracking and select TrackIR.

**Reply by htorquato-dev** [reply]
2025 already and it worked for me!! Thanks for share!
